sycl/unittests/pi/cuda/test_base_objects.cpp

```diff
@@ -118,6 +119,52 @@ TEST_F(DISABLED_CudaBaseObjectsTest, piContextCreatePrimary) {
       PI_SUCCESS);
 }
 
+TEST_F(DISABLED_CudaBaseObjectsTest, piContextCreatePrimaryFalse) {
```

Comment:
Feel free to ignore: probably it is a good spot to outline some common code into helper functions, or use [value-parameterized tests](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#value-parameterized-tests)

Reply:
I concur, and that is something in preparation.

We need this fix quickly though so I only prepared a testcase (and tested locally by hand) and did not want to introduce extra work that slows this PR down with refactoring of the test approach.